Accompany/Views/AddNoteCellOneTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AddNoteTableViewCell.swift
+//  Accompany
+//
+//  Created by Kelbin David on 2022-02-10.
+//
+
+import UIKit
+import SnapKit
+
+class AddNoteCellOneTableViewCell: UITableViewCell {
```

Comment:
Please refactor the name of this class and file to "TodoBasicInfoTableViewCell", it's more semantic and readable.
```suggestion
class TodoBasicInfoTableViewCell: UITableViewCell {
```